app-frontend/src/app/pages/projects/edit/annotate/annotate.html

```diff
@@ -1,111 +1,113 @@
-<!-- Annotate Toolbar Component -->
-<rf-annotate-toolbar map-id="edit"
-                     disable-toolbar-action="$ctrl.disableToolbarAction"
-                     on-shape-creating="$ctrl.onShapeCreating(isCreating)"
-                     on-shape-created="$ctrl.onShapeCreated(shapeLayer)">
-</rf-annotate-toolbar>
-<!-- Annotate Toolbar Component -->
+<div class="sidebar annotate">
+  <!-- Annotate Toolbar Component -->
+  <rf-annotate-toolbar map-id="edit"
+                       disable-toolbar-action="$ctrl.disableToolbarAction"
+                       on-shape-creating="$ctrl.onShapeCreating(isCreating)"
+                       on-shape-created="$ctrl.onShapeCreated(shapeLayer)">
+  </rf-annotate-toolbar>
+  <!-- Annotate Toolbar Component -->
```

Comment:
I don't think we need these comments in the html - the component titles / style classes are pretty self explanatory at this level